draft-handrews-jri.md

```diff
@@ -0,0 +1,536 @@
+
+Discussion of this draft takes place in the [GitHub repository](https://github.com/json-schema-org/referencing) and on the [JSON Schema Slack server](https://json-schema.org/slack).
+
+--- middle
+
+# Introduction
+
+JSON Reference and Identification (JRI) offers a modular set of features to enable interoperable support of identification and referencing across a wide range of formats.  These features are necessary to create data formats across a set of linked documents, but are often incidental to the primary functionality of a data format specification.  Offloading these features to common libraries and tools based on a standardized approach allows data format implementations to focus on the format's purpose with minimal need to understand complex and potentially multi-document structures.
+
+Standalone JRI use cases include a variety of document transformations, transparent in-memory representations of linked documents, and locating, caching, and serving resources identified within or referenced from a set of documents.
+
+While {{?RFC8288}} web linking solves this problem in a more general way, a simplified, concise, and self-contained linking and identification system is better-suited to many specifications.  This allows those specifications to avoid complexity, and to avoid constraining their syntax to match any one of the many competing web linking-enabled JSON formats.
+
+To support a variety of use cases, JRI supports trade-offs between generic interoperability and tight integration with the context in which it is used.  JRI defines an interoperable subset of its features for standalone use, while encouraging specifications that need reference and/or identification features to incorporate as much or as little of JRI as suits their needs.  JRI also defines mechanisms for resources to indicate their JRI usage, allowing generic or standalone implementations to understand how to safely process a document without knowledge of its underlying data format.
+
+## Notational Conventions
+
+{::boilerplate bcp14+}
+
+The terms "primary resource" and "secondary resource" in this specification are to be interpreted as in {{!RFC3986, Section 3.5}}.
+
+Due to the distinction that this specification makes between documents and resources, the term "same-resource reference" is to be interpreted in the same way as "same-document reference" as defined in {{!RFC3986, Section 4.4}}.  The term "same-document reference" is not used in this specification.
```

Comment:
@DavidBiesack I'm a little unclear on when something goes under "Notation conventions" and when it goes under "Definitions" - I certainly have no objection to moving it, I just have no idea which is correct.  Terms that aren't redefined definitely go in "Notation conventions" and entirely new terms definitely go in "Definitions".  Maybe I should just mention it in both?

Reply:
since the notations section is so small, mentioning in both seems redundant. I prefer to see _new_ terms defined in Definitions, use of existing concepts/conventions in the prior, and this feels like a term/definition